### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by GraceAbike**
### In this pull request I did the followings:
 - [x] [The Leaderboard API](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md)
 - [x] Created a new Game with the API using [Testfully App](https://docs.testfully.io/):
`{ "result": "Game with ID: OOyBSMfV40z7yA2OR8pe added." }`
 - [x] Implemented the `Refresh` button that will get all scores of the created game from the API.
- [x]  Implemented the form `Submit` button that saves a score for the created game (sending data to the API).
 - [x] Used proper `ES6` syntax.
 - [x] Used ES6 `modules` to write modular JavaScript.
- [x] Used `arrow functions` instead of the `function` keyword.
 - [x] Used `async` and `await` JavaScript features to the API.
 - [x] Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- [x] Made sure of using correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] Made sure of documenting the project [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
## Appreciation
Thanks for the review 🙂

### CODE REVIEW

File: src/modules/configuration.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const gameKey = 'OOyBSMfV40z7yA2OR8pe';
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/OOyBSMfV40z7yA2OR8pe/scores';
+
+const apiEndPoint = `${baseURL}/games/${gameKey}/scores`;
```
Comment (leonardodiasb) [inline src/modules/configuration.js]: - [ ] It seems that you are passing your 'gameKey' and other URL information twice in your 'apiEndPoint' and this is preventing the user to connect with your application. Please fix this issue.
 -'Server Down':
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @GraceAbike, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of Gitflow. ✔️ 
 - Good use of ES6 best practices. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/configuration.js (hunk 1 above)
Comment (GraceAbike) [reply src/modules/configuration.js]: @leonardodiasb thanks for the comments, I will fix it now

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @GraceAbike 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️
 - Correct use of Gitflow. ✔️
 - Good use of ES6 best practices. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**